store/dataStore.ts

```diff
@@ -30,4 +65,26 @@ export const useDataStore = create<DataStore>((set) => ({
       }),
     ),
   resetUser: () => set({ user: initialState.user }),
+  updateNFLTeams: (teamName, teamLogo) =>
+    set(
+      produce((state: IDataStoreState) => {
+        state.NFLTeams.teamName = teamName;
+        state.NFLTeams.teamLogo = teamLogo;
+      }),
+    ),
+    updateUserWeeklyPick: (id, weekNumber) => 
+    set(
+      produce((state: IDataStoreState) => {
+        state.user.id = id;
+        state.userWeeklyPick.weekNumber = weekNumber
+      })
+    ),
+    updateWeeklyPicks: (gameId, gameWeekId, userResults) => 
+    set(
+      produce((state: IDataStoreState) => {
+        state.weeklyPicks.gameId = gameId;
```

Comment:
w: find a way to set these in 1 object call.

Reply:
Updated in this object call:

`
updateWeeklyPicks: ({
    gameId,
    gameWeekId,
    userResults,
  }: IWeeklyPicks): void =>
    set( 
      produce((state: IDataStoreState) => {
        state.weeklyPicks.gameId = gameId;
        state.weeklyPicks.gameWeekId = gameWeekId;
        state.weeklyPicks.userResults = userResults;
      })
      `